packages/sx05re/emuelec-ports/openjazz/package.mk

```diff
@@ -0,0 +1,20 @@
+PKG_NAME="openjazz"
+PKG_VERSION="a34e79dba5756000ad3796efae61a57be4a46496"
+PKG_SHA256="fe76aca99a319ee70c1eaeec17a935880c7a73f02e42ac1917e581062ea38e67"
+PKG_ARCH="aarch64"
+PKG_LICENSE="GPL-2.0-or-later"
+PKG_SITE="http://www.alister.eu/jazz/oj/"
+PKG_URL="https://github.com/sana2dang/openjazz/archive/a34e79dba5756000ad3796efae61a57be4a46496.tar.gz"
```

Comment:
to make it easy to update please use `https://github.com/sana2dang/openjazz/archive/$PKG_VERSION.tar.gz`